src/app/shared/chips/chips.component.ts

```diff
@@ -0,0 +1,89 @@
+import { ChangeDetectorRef, Component, EventEmitter, Input, OnChanges, Output, SimpleChanges, } from '@angular/core';
+
+import { NgbTooltip } from '@ng-bootstrap/ng-bootstrap';
+import { SortablejsOptions } from 'angular-sortablejs';
+
+import { Chips } from './models/chips.model';
+import { ChipsItem } from './models/chips-item.model';
+import { UploaderService } from '../uploader/uploader.service';
+
+@Component({
+  selector: 'ds-chips',
+  styleUrls: ['./chips.component.scss'],
+  templateUrl: './chips.component.html',
+})
+
+export class ChipsComponent implements OnChanges {
```

Comment:
It's also not clear to me what Chips are in this context.

Reply:
`ChipsComponent` is used to render and handle (add/remove/sort) chip elements and can be used in a form :
![chips](https://user-images.githubusercontent.com/2486489/42310021-a03d4a42-803a-11e8-93a0-52014557f176.JPG)

Reply:
@atarix83 : this is something that comments in the component could help with.  Perhaps we need to describe what a "Chip" element is, possibly even with links for examples.  I admit, I didn't understand the work "chip" here either. But, I see it's a common term: https://www.w3schools.com/howto/howto_css_contact_chips.asp